sync_into_benchling/sync_into_benchling.py

```diff
@@ -84,7 +86,7 @@ def find_or_create_chain_with_aa_sequence(
     if existing_registered_chain_json is not None:
```

Comment:
I think we can just try to create-into-registry in all cases, and then rely on the unique constraint to return a 409 CONFLICT if it exists. Otherwise we risk having a race condition here (between checking if it exists, and creating it).

Reply:
I thought api_get would be a synchronous call. I don't think there is a race condition between check if the AA sequence exists in the registry and creating it.